lib/9pfs/9pfs.h

```diff
@@ -39,63 +39,150 @@
 
 #include <vfscore/prex.h>
 
-/*
- * Currently supports only the 9P2000.u variant of the protocol.
+/**
+ * enum uk_9pfs_proto - Protocol version;
+ * currently only the 9P2000.u variant of the protocol is supported
  */
 enum uk_9pfs_proto {
+	/*
+	 * Some extensions and minor adjustments are added to
+	 * the basic protocol for Unix systems
+	 */
 	UK_9P_PROTO_2000U,
+	/*
+	 * Some extensions and minor adjustments are added to
+	 * the basic protocol for Linux systems
+	 */
 	UK_9P_PROTO_2000L,
+	/*
+	 * The number of existent protocols;
+	 * in this case, two (UK_9P_PROTO_2000U and UK_9P_PROTO_2000L)
+	 */
 	UK_9P_PROTO_MAX
 };
 
+/**
+ * struct uk_9pfs_mount_data - An entry containing the necessary data
+ * for mounting the filesystem
+ */
 struct uk_9pfs_mount_data {
-	/* 9P device. */
+	/* 9P device */
 	struct uk_9pdev		*dev;
-	/* Wanted transport. */
+	/* Wanted transport */
 	struct uk_9pdev_trans	*trans;
-	/* Protocol version used. */
+	/* Protocol version used */
 	enum uk_9pfs_proto	proto;
-	/* Username to attempt to mount as on the remote server. */
+	/* User name to attempt to mount as on the remote server */
 	char			*uname;
-	/* File tree to access when offered multiple exported filesystems. */
+	/*
+	 * File tree to access when the server is
```

Comment:
File tree? It's unclear what this is.

Reply:
From what I understand `aname` is the name of the file system that we want to use. So file tree should be the file system.